server.js

```diff
@@ -1,9 +1,61 @@
+// implement your API here
 const express = require('express');
+const dbU = require('./data/helpers/userDb');
 const morgan = require('morgan');
-const helmet = require('helmet');
-const cors = require('cors');
-
+// const helmet = require('helmet');
+// const cors = require('cors');
 const server = express();
+
+//++++++++++++++++++++++++ 
+// server port
+
 const PORT = 5050;
 
-server.listen(PORT, () => console.log(`The Server is listening on port ${PORT}`));
\ No newline at end of file
+
+//++++++++++++++++++++++++ 
+// golbal middleware and addins
+//+++++++++++++++++++++++
+
+
+// server.use(helmet()); // provides basic security
+
+server.use(express.json()); // json bdy parser
+
+server.use(morgan('short')); // logger/tracer of activity
+
+// server.use(cors()); //cross domain issues
+
+
+//++++++++++++++++++++++++++++++++++++++++++
+// get endpoints
+//++++++++++++++++++++++++++++++++++++++++++++
+
+//   server.get("/api/posts/:userId", async (req, res) => {
```

Comment:
Remember that we're doing users today, not posts (although it looks like you're getting a specific user here, so this is just the wrong url and that's all)!